rhoas-cli-kafka/README.adoc

```diff
@@ -0,0 +1,265 @@
+////
+START GENERATED ATTRIBUTES
+WARNING: This content is generated by running npm --prefix .build run generate:attributes
+////
+
+
+:community:
+:imagesdir: ./images
+:product-version: 1
+:product-long: Application Services
+:product: App Services
+:registry-product-long: OpenShift Service Registry
+:registry: Service Registry
+// Placeholder URL, when we get a HOST UI for the service we can put it here properly
+:service-url: https://console.redhat.com/beta/application-services/streams/
+:registry-url: https://console.redhat.com/beta/application-services/service-registry/
+:property-file-name: app-services.properties
+:rhoas-version: 0.29.0
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
```

Comment:
So we need three new URLs here. One each for the cli getting starteds and one for the cli install?

Reply:
Who will be the best to ask how this works?